Project Plan

```diff
@@ -0,0 +1,14 @@
+Topic - Correlation between Oil prices, currency valuation and consumer prices
+
+Questions - How does the cost of oil cause fluctuations in the valuation of various currencies and how does it affect the cost of items for consumers.
+        Side Note - The scope will be from January 2009 to December 2019 due to the Covid pandemic's affect on data between March 2020 and December 2021 and 
+        Russian invasion of Ukraine causing an oil panic
+        
+Sample preliminary answer contained within PowerBI file
+
+Data sources are government and economic online sources
```

Comment:
Not sure if you've already used it, but worldbank.org is a good data source too since you mentioned currency valuation and cost of items for consumers.